backend/dataall/core/environment/db/models.py

```diff
@@ -67,6 +67,7 @@ class EnvironmentParameter(Base):
     value = Column('paramValue', String, nullable=True)
 
     def __init__(self, env_uri, key, value):
+        super().__init__()
```

Comment:
Just for curiosity, nothing happens if we leave it right? although Base has no init

Reply:
Nope, but my PyCharm was so persistent that I had to fix it